src/backend/looped/world/groundsnow.cpp

```diff
@@ -0,0 +1,29 @@
+#include "backend/looped_command.hpp"
+#include "fiber_pool.hpp"
+#include "gta_util.hpp"
+#include "script.hpp"
+#include "services/tunables/tunables_service.hpp"
+
+namespace big
+{
+	class ground_snow : looped_command
+	{
+		using looped_command::looped_command;
+
+		virtual void on_enable() override
+		{
+			g_tunables_service->set_tunable<bool>(RAGE_JOAAT("TURN_SNOW_ON_OFF"), true);
+		}
```

Comment:
You may want to move this to on_tick, as it's possible the tunable service will overwrite this the next time they refresh, which is usually done during transition or session change. That is your goal I'm assuming, since you are saving it to a variable called ground_snow.

Reply:
Haven't looked into if this specific tunable gets refreshed anytime, but now you mention it you are also able refresh other player's tunables through a script event so I'll move it to on_tick, appreciate the heads-up.